en/ui-system/components/editor/safearea.md

```diff
@@ -0,0 +1,13 @@
+# SafeArea component reference
+
+This component is used to adjust the layout of current node to respect the safe area of a notched mobile device such as the iPhone X.
+
+It is typically used for the top node of the UI interaction area. For specific usage, refer to the [SafeArea](https://github.com/cocos-creator/test-cases-3d/tree/master/assets/cases/ui/20.safe-area) in the example-cases.
+
+The concept of safe area is to give you a fixed inner rectangle in which you can safely display content that will be drawn on screen.
+
+You are strongly discouraged from providing controls outside of this area. But your screen background could embellish edges.
```

Comment:
You are strongly discouraged from providing controls outside of this area. But your screen background could embellish edges.
-->
It is strongly discouraged from providing controls outside of this area. But the screen background could embellish edges.